pkg/k8shandler/reconciler.go

```diff
@@ -68,14 +75,41 @@ func Reconcile(requestCluster *logging.ClusterLogging, requestClient client.Clie
 
 	// Reconcile Collection
 	if err = clusterLoggingRequest.CreateOrUpdateCollection(); err != nil {
-		return fmt.Errorf("Unable to create or update collection for %q: %v", clusterLoggingRequest.Cluster.Name, err)
+		msg := fmt.Sprintf("Unable to create or update collection for %q: %v", clusterLoggingRequest.Cluster.Name, err)
+		clusterLoggingRequest.Cluster.Status.Conditions.SetCondition(condDegraded(logging.ReasonInvalid, msg))
+		log.Error(err, "Cluster Logging transition to Degraded state: "+msg)
 	}
 
 	// Reconcile Metrics Dashboards
 	if err = clusterLoggingRequest.CreateOrUpdateDashboards(); err != nil {
-		return fmt.Errorf("Unable to create or update metrics dashboards for %q: %w", clusterLoggingRequest.Cluster.Name, err)
+		msg := fmt.Sprintf("Unable to create or update metrics dashboards for %q: %v", clusterLoggingRequest.Cluster.Name, err)
+		clusterLoggingRequest.Cluster.Status.Conditions.SetCondition(condDegraded(logging.ReasonMissingResource, msg))
+		log.Error(err, "Cluster Logging transition to Degraded state: "+msg)
+	}
+
+	spec := &logging.ClusterLogForwarderSpec{}
+	status := &logging.ClusterLogForwarderStatus{}
+	clusterLoggingRequest.verifyInputs(spec, status)
```

Comment:
Trying to follow Alan's [comment](https://github.com/openshift/cluster-logging-operator/pull/993#discussion_r622433383) and yes you are right we aready do it [here](https://github.com/openshift/cluster-logging-operator/blob/master/pkg/k8shandler/forwarding.go#L123-L134), but if i understand correct `NormalizeForwarder` don't call during collector reconciliation. Please, correct me if i wrong, becpuse logic of recocilation not fully cleare for me at the moment.

Reply:
The status from CLF are displayed on CLF.  IMO we should refrain from duplicating them on CL  I think its fair to distinguish CL from CLF and it may be possible that CL is degraded while CLF is not and visa versa